kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/web/struts/PaymentRequestAction.java

```diff
@@ -0,0 +1,96 @@
+import org.kuali.kfs.module.purap.document.PaymentRequestDocument;
+import org.kuali.kfs.module.purap.document.PurchaseOrderDocument;
+import org.kuali.kfs.module.purap.document.web.struts.PaymentRequestForm;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.config.property.ConfigurationService;
+import org.kuali.rice.kns.question.ConfirmationQuestion;
+
+import edu.arizona.kfs.fp.service.DisbursementVoucherInvoiceService;
+import edu.arizona.kfs.module.purap.PurapConstants;
+import edu.arizona.kfs.module.purap.PurapKeyConstants;
+import edu.arizona.kfs.sys.KFSConstants;
+import edu.arizona.kfs.sys.KFSKeyConstants;
+
+public class PaymentRequestAction extends org.kuali.kfs.module.purap.document.web.struts.PaymentRequestAction {
+    @SuppressWarnings("unused")
+    private static Logger LOG = Logger.getLogger(PaymentRequestAction.class);
+
+    private static transient volatile DisbursementVoucherInvoiceService disbursementVoucherInvoiceService;
+    private static transient volatile ConfigurationService configurationService;
+
+    private static DisbursementVoucherInvoiceService getDisbursementVoucherInvoiceService() {
+        if (disbursementVoucherInvoiceService == null) {
+            disbursementVoucherInvoiceService = SpringContext.getBean(DisbursementVoucherInvoiceService.class);
+        }
+        return disbursementVoucherInvoiceService;
+    }
+
+    private static ConfigurationService getConfigurationService() {
+        if (configurationService == null) {
+            configurationService = SpringContext.getBean(ConfigurationService.class);
+        }
+        return configurationService;
+    }
+
+    @Override
+    protected ActionForward performDuplicatePaymentRequestAndEncumberFiscalYearCheck(ActionMapping mapping, ActionForm form, HttpServletRequest request, HttpServletResponse response, PaymentRequestDocument paymentRequestDocument) throws Exception {
+        ActionForward forwardIfDuplicatePREQ = super.performDuplicatePaymentRequestAndEncumberFiscalYearCheck(mapping, form, request, response, paymentRequestDocument);
+        if (forwardIfDuplicatePREQ != null) {
+            return forwardIfDuplicatePREQ;
+        }
+
+        Object question = request.getParameter(KFSConstants.QUESTION_INST_ATTRIBUTE_NAME);
+        Object buttonClicked = request.getParameter(KFSConstants.QUESTION_CLICKED_BUTTON);
+        if (PurapConstants.PREQ_DUPLICATE_DV_QUESTION.equals(question)) {
+            if (ConfirmationQuestion.NO.equals(buttonClicked)) {
+                paymentRequestDocument.getFinancialSystemDocumentHeader().setWorkflowDocumentStatusCode(PurapConstants.PaymentRequestStatuses.APPDOC_INITIATE);
+                String errorText = getConfigurationService().getPropertyValueAsString(PurapKeyConstants.MESSAGE_PREQ_DUPLICATE_DV_INVOICE);
+                // GlobalVariables.getMessageMap().putError(PurapPropertyConstants.INVOICE_NUMBER, PurapKeyConstants.MESSAGE_PREQ_DUPLICATE_DV_INVOICE);

```

Comment:
Please remove commented out code and line above it (line 60) as this also appears unneeded.